README.md

```diff
@@ -13,3 +13,21 @@ Make it sticky
 
 Since version 1.0.6. there is an option in the plugin settings, to create a global sticky instance.
 It will be shown in all course main pages.
+
+Upgrade to v2
```

Comment:
Mache bitte "Upgrade to Version 2" daraus. Es ist nicht ganz ersichtlich, warum innerhalb des Textes "version" ausschreibst aber dann in einer Überschrift nicht.